pkg/components/exporter/exporter.go

```diff
@@ -249,6 +218,40 @@ func NewConfigMap(d components.Detail, k *v1alpha1.KeplerInternal) *corev1.Confi
 	deployment := k.Spec.Exporter.Deployment.ExporterDeploymentSpec
 	bindAddress := "0.0.0.0:" + strconv.Itoa(int(deployment.Port))
 
+	modelConfig := ""
+	if k.Spec.Estimator != nil {
+		modelConfig = estimator.ModelConfig(k.Spec.Estimator)
+	}
+
+	exporterConfigMap := k8s.StringMap{
+		"KEPLER_NAMESPACE":                  k.Namespace(),
+		"KEPLER_LOG_LEVEL":                  "1",
+		"METRIC_PATH":                       "/metrics",
+		"BIND_ADDRESS":                      bindAddress,
+		"ENABLE_GPU":                        "true",
+		"ENABLE_QAT":                        "false",
+		"ENABLE_EBPF_CGROUPID":              "true",
+		"EXPOSE_HW_COUNTER_METRICS":         "true",
+		"EXPOSE_IRQ_COUNTER_METRICS":        "true",
+		"EXPOSE_KUBELET_METRICS":            "true",
+		"EXPOSE_CGROUP_METRICS":             "true",
+		"ENABLE_PROCESS_METRICS":            "false",
+		"CPU_ARCH_OVERRIDE":                 "",
+		"CGROUP_METRICS":                    "*",
+		"REDFISH_PROBE_INTERVAL_IN_SECONDS": "60",
+		"REDFISH_SKIP_SSL_VERIFY":           "true",
+		"MODEL_CONFIG":                      modelConfig,
+	}
+
+	ms := k.Spec.ModelServer
+	if ms != nil {
+		if ms.Enabled {
+			exporterConfigMap["MODEL_SERVER_ENABLE"] = "true"
+		}
+		modelServerConfig := modelserver.ModelServerConfigForClient(k.ModelServerDeploymentName(), k.Spec.ModelServer)
```

Comment:
```suggestion
		modelServerConfig := modelserver.ConfigForClient(k.ModelServerDeploymentName(), k.Spec.ModelServer)
```
How about renaming `modelserver.ModelServer<X>` to `modelserver.<X>` to avoid stuttering ?

Reply:
I wonder if `k.Spec.ModelServer.ClientConfig()` is  better since all the function only requires the spec from  Modelserver to compute client config.

Reply:
Yes..it is the config for client of the model server (i.e., exporter/estimator container).

Reply:
@sthaha Please confirm whether the change resolved?